src/llmcompressor/transformers/compression/sparsity_config.py

```diff
@@ -82,6 +90,11 @@ def from_pretrained(
         :param state_dict: optional state_dict to replace that in model, used for
         gathering global FSDP model info
         :param compress: whether or not to compress the model on disk
+        :param quantization_format: the quantization compression format being used
```

Comment:
Does this value have to be nullable? If so, could you explain what the significance of the null value is (ie, infer from config)

Reply:
None represents no quantization!